### ISSUE TITLE
NPE in org.roaringbitmap.FastAggregation.and(Iterator<RoaringBitmap>) if Iterator has exactly one element

### ISSUE THREAD

**Issue by blasd** [lemire/RoaringBitmap#39]
If the Iterator is empty has at least 2 elements, the result is OK. If it has exactly one element, one get an NPE. Same behavior for the array version of and.

```java
java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
	at java.util.ArrayList.rangeCheck(ArrayList.java:635)
	at java.util.ArrayList.get(ArrayList.java:411)
	at org.roaringbitmap.FastAggregation.and(FastAggregation.java:70)

**Comment by lemire** [comment]
Verified. Thanks. Bug will be fixed in upcoming release.

**Comment by lemire** [comment]
Issue fixed in 0.4.11. Please reopen if you disagree.
